docs/sources/whatsnew/whats-new-next/index.md

```diff
@@ -18,6 +18,19 @@ weight: -37
 
 Welcome to Grafana Cloud! Read on to learn about the newest changes to Grafana Cloud.
 
+## Content outline
+
+<!-- Thanos Karachalios -->
+<!-- OSS,Enterprise -->
+
+October 12, 2023
+
+_Generally Available in Grafana Cloud_
+
+Introducing Content Outline in Grafana Explore. We recognized the challenges of complex mixed queries, as well as, lengthy logs and traces results, leading to time-consuming navigation and the loss of context. Content outline is our first step towards seamless navigation from log lines to traces and back to queries ensuring quicker searches while preserving context. Experience efficient, contextual investigations with this update in Grafana Explore. To learn more, refer to the [Content outline documentation](), as well as the following video demo.
```

Comment:
Generally we don't capitalize features like this that don't show up in the UI with the exact same name, but I'll leave it for now. Something to keep in mind as you're naming features.